### DISCUSSION TITLE
Configuring console log severity levels

### DISCUSSION THREAD

**Discussion by rbraggaar** [directus/directus#10839] [Feature Requests]
Would it be possible to turn off the console logging of certain severity levels, for example debug/info (see screenshot attached)?
Ideally through a setting in the .env file.

Of course it is fine to display the Directus logo/credits, but I would like to keep the log clean and only display errors (in case of any).

![image](https://user-images.githubusercontent.com/16044244/148036525-8a1b35c5-48b2-4619-a134-7eb671477e5d.png)

**Comment by azrikahar** [comment]
Currently it is not possible since the App won't have access to environment variables to do that. We can only control the severity level on the API side, via `LOG_LEVEL`.

We can technically make it so that it'll call an endpoint and turn off/on based on a particular value, but it might be an overkill solution/workaround here 🤔

**Reply by rbraggaar** [reply]
Thanks, I understand. A setting like on the API side was what I was looking for on the App side as well.
What do you think about adding it to the project settings as an option?

**Reply by azrikahar** [reply]
That would surely be a possible approach, though it might not fit the [80/20 rule](https://docs.directus.io/contributing/introduction/#feature-requests) to fit in core. 

However I'm also wondering if we can come up with a workaround for you in the meantime, but I can't think of any at the moment as it's rather "hardcoded" here:

https://github.com/directus/directus/blob/4c1ac51e8b1257f0b8575554dde12c730ca76014/app/src/main.ts#L56-L65

**Reply by nickrum** [reply]
@rbraggaar In case you just want to keep things clean for yourself while debugging, most browsers already support setting the log level inside the console window.

**Reply by rijkvanzanten** [reply]
There's no way to configure that from within Directus. @nickrum is right, the way to do this is within the devtools itself:

<img width="768" alt="CleanShot 2022-01-04 at 10 25 17@2x" src="https://user-images.githubusercontent.com/9141017/148082282-b7eea037-a7ba-4f44-9003-72670c6f84b7.png">

**Reply by rbraggaar** [reply]
Thanks, I'm already using that myself indeed.
Just thought things would look a little bit neater since I'm running Directus in production as the back-end for one of my apps.
(for users who might pop open the dev tools themselves and get presented with a word art and details that don't matter to them)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
